PocketKit/Sources/PocketKit/Home/Cells/Carousel/RecentSavesCellConfiguration.swift

```diff
@@ -0,0 +1,71 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import UIKit
+import Textile
+import Localization
+import SharedPocketKit
+
+struct RecentSavesCellConfiguration: HomeCarouselCellConfiguration {
```

Comment:
At somepoint we should talk about these cell configs, feels like all the logic here duplicates a lot.

Reply:
Totally agree